modules/gitops-registering-an-oauth-client.adoc

```diff
@@ -0,0 +1,31 @@
+// Module is included in the following assemblies:
+//
+// * 
+
+[id="registering-an-oauth-client_{context}"]
+= Registering an OAuth client
+
+If you need an additional OAuth client to manage authentication for your OpenShift Container Platform cluster, you can register one.
+
+.Procedure
+
+* To register your client:
++
+[source,terminal]
+----
+$ oc create -f <(echo '
+kind: OAuthClient
+apiVersion: oauth.openshift.io/v1
+metadata:
+ name: keycloak-broker <1>
+secret: "..." <2>
+redirectURIs: <3>
+"https://keycloak-keycloak.apps.dev-svc-4.7-020201.devcluster.openshift.com/auth/realms/myrealm/broker/openshift-v4/endpoint" 
+grantMethod: prompt <4>  
+')
+----
+
+<1> The name of the OAuth client is used as the `client_id` parameter when making requests to <namespace_route>/oauth/authorize and <namespace_route>/oauth/token.
+<2> The `secret` is used as the client_secret parameter when making requests to <namespace_route>/oauth/token.
+<3> The `redirect_uri` parameter specified in requests to <namespace_route>/oauth/authorize and <namespace_route>/oauth/token must be equal to or prefixed by one of the URIs listed in the `redirectURIs` parameter value. 
+<4> The `grantMethod` is used to determine what action to take when this client requests tokens and has not yet been granted access by the user. Specify auto to automatically approve the grant and retry the request, or prompt to prompt the user to approve or deny the grant.
```

Comment:
Could you leave one "prompt" out and just prompt the user maybe? Not sure what makes sense here, but I stumbled on it while reading.

Reply:
`prompt` should be in backticks, it's an option to specify